requirements-runtime.txt

```diff
@@ -1,4 +1,4 @@
 msgpack
 ndindex>=1.4
-numpy>=1.20.3
+oldest-supported-numpy
```

Comment:
I think for runtime `numpy<2` is a better option (`oldest-supported-numpy` is mainly for building purposes).